crates/ty_python_semantic/mdtest.py

```diff
@@ -165,9 +181,9 @@ def _print_trimmed_cargo_test_output(self, output: str, test_name: str) -> None:
 
             print(line)
 
-    def watch(self) -> Never:
+    def watch(self):
```

Comment:
isn't the type annotation still accurate? :P

Reply:
I think it was never accurate. Both ty and pyright complained. There's no way for a type checker to see that the loop runs forever.

Reply:
You could add `assert False, "Should never get here"` at the bottom of the function to shut them up, I guess :P